src/Text/LLVM/PP.hs

```diff
@@ -881,8 +851,8 @@ ppValMd :: Fmt ValMd
 ppValMd = ppValMd' ppLabel
 
 ppDebugLoc' :: Fmt i -> Fmt (DebugLoc' i)
-ppDebugLoc' pp dl = (if cfgUseDILocation ?config then "!DILocation"
-                                                 else "!MDLocation")
+ppDebugLoc' pp dl = (if llvmVer > llvmV3_6 then "!DILocation"
```

Comment:
I'd prefer checking `llvmVer >= llvmV3_7` here so that you don't have to add 1 to the version when reading it.